### ISSUE TITLE
Release a 2nd wave of Backport packages

### ISSUE THREAD

**Issue by potiuk** [apache/airflow#10014]
**Description**

I think we are really close to release the second wave of Backport Packages. We accumulated a number of changes already and few more are about to be completed so that we are, I think, close to release the second wave. 

** What will be in those packages **

Everything that is currently merged to the master branch in "providers" package will be included.

Those are open issues that we know we should merge before though:

- [x] ~Enabling `cncf.kubernetes` provider after 1.10.11/1.10.12 backported changes and spark-on-kubernetes operator~ - this is deferred to the next backport release as per #11298 
- [x] Implementing Impersonation for all applicable Google operators (@olchas works on it with Hooks already merged)
- [x] DataProc operator improvements (@turbaszek works on it in #9593 )
- [x] Improve idempotency of BigQueryInsertJobOperator (@turbaszek works on it in #9590)
- [x] Release 1.10.12 of Airflow as it fixes some teething problems with Kubernetes refactor from 1.10.11
- [x] Fix BigQuery operators problem with serialization #10171
- [x] AWS SES #10004
- [x] Fix the DLP operators.
- [x] ~finalize and implement the #10822 allowing to release 1.10.13 with support to auto-dicover provider capabilities such as connections and extra links~ deferred to another release
- [x] ~Merge #10488 with Kerberos on Presto change.~ deferred to the next release
- [x] ~Breaking changes: Email operator moved to Core: we are going to address it in 1.10.13 by adding email provider there and releasing provider package via #11178~
 
We have to make the following adjustments to the process:

- [x] make sure that we only release the packages that have some changes #back #10375
- [x] make sure to add changes from the previous releases - not only commits but add mechanism to add
       backwards-incompatible changes akin to UPDATING.md  #10376

Update:
The second wave is delayed a bit (but coming soon). The reason is two-fold

* the cncf.kubernetes package has a problem that we are solving
* we might try to get the mechanism of separate providers packages and "discovery" mechanism before we build the packages - see the #10822 and #10823 

**Use case / motivation**

We want the users of airflow 1.10 to benefit from fixes and improvements in the latest packages.

**Comment by potiuk** [comment]
@kaxil @turbaszek @dimberman @mik-laj @olchas (and other @apache/airflow-committers who are interested).

 I am going for holidays today, and won't be available for a week or so, but I think this one needs some love while I am away, and maybe discussion if there are any more things we want to include in backport packages.

I think there are few small fixes needed in backport release scripts/process to make sure the "2nd wave" works on top of the "1st wave" smoothly but they're just minor adjustments (filtering out some commits from the reports and filtering out packages that are unaffected + adding the new ones that we will add).

I am happy to take care about it next week when I am back :)

**Comment by potiuk** [comment]
cc: @r-richmond

**Comment by mik-laj** [comment]
We should fix this issue before this packages is released.
https://github.com/apache/airflow/issues/10171

**Comment by JeffryMAC** [comment]
please also include AWS SES https://github.com/apache/airflow/pull/10004

**Comment by potiuk** [comment]
> please also include AWS SES #10004

Done :)

**Comment by guptamyr** [comment]
Hi, could you please also make sure to include this new operator for AWS SageMaker: https://github.com/apache/airflow/pull/9594. Thanks!

**Comment by potiuk** [comment]
Yep. It's alrready merged so it will be inclued automagically :). Baically we release everything that is now in master.

**Comment by mik-laj** [comment]
We have breaking change between backport packages. - EmailOperator has been moved to core.

**Comment by mik-laj** [comment]
Before releasing a new package, we should fix the operators for DLP.

**Comment by bodak** [comment]
Are you planning to include GKEStartPodOperator? 

https://github.com/apache/airflow/blob/master/backport_packages/refactor_backport_packages.py#L602
Removed in PR: https://github.com/apache/airflow/pull/7908

**Comment by mik-laj** [comment]
Yes. It is planned.

**Comment by potiuk** [comment]
I am planning to cut the 2nd Backport package release candidate this evening. So please try to merge everything from "providers" if you want this to be part of that release.

**Comment by guptamyr** [comment]
Hi, 

Has the 2nd wave of backport packages been released or is there a new ETA?

**Comment by potiuk** [comment]
Sorry - I should have updated it. The second wave is delayed a bit (but coming soon). The resason is two-fold
* the `cncf.kubernetes` package has a problem that we are solving
* we might try to get the mechanism of separate providers packages and "discovery" mechanism before we build  the packages

**Comment by mik-laj** [comment]
I would like this change to be included. I am working on doing a rebase.
https://github.com/apache/airflow/pull/10488

**Comment by mik-laj** [comment]
RC1 packages have been released. Feel free to test and report problems. 
https://lists.apache.org/thread.html/r4dfc18d2376601667efe7dfaf0bf1c07b840f9bcd9f6311d48cb3c64%40%3Cdev.airflow.apache.org%3E

**Comment by potiuk** [comment]
The Voting passed. We are going to release everythiong except "cncf.kubernetes" and we are going to have a follow-up release of "google" and "cncf.kubernetes" providers after the #11298 and #11299 has been fixed. We are going also to have some follow-up release after we release 1.10.13 with #10822, #10823 and #11178. We are also likely to release Presto operator as well with added  Kerberos support then #10488.

**Comment by potiuk** [comment]
Backport released.
